Formula/lcm.rb

```diff
@@ -25,18 +25,18 @@ class Lcm < Formula
   depends_on "python@3.9"
 
   def install
+    # Adding RPATH in #{lib}/lua/X.Y/lcm.so and some #{bin}/*.
     args = std_cmake_args + %W[
+      -DCMAKE_INSTALL_RPATH=#{lib}
```

Comment:
Missing RPATH in `#{bin}/*` can use `#{rpath}`, but the `#{lib}/lua/X.Y/lcm.so` is nested so `#{rpath}` may not point to correct relative path. So, just using `#{lib}`.